lib/sequelize.d.ts

```diff
@@ -144,6 +154,33 @@ export interface OperatorsAliases {
     [K: string]: symbol
 }
 
+/**
+ * Final config options generated by sequelize.
+ */
+export interface Config {
+    readonly database: string
```

Comment:
Why `readonly`?

Reply:
I'm pretty sure that changing this at runtime causes all sorts of issues, looking at the source code this looks like semi internal, despite not being _ prefixed.